### PR TITLE
Upgraded to jest 25.2.0

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

jest has now been upgraded to the latest version, 25.2.0

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

Television message saying they are performing a station test of the broadcast sytem

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ module.exports = {
         },
         useBuiltIns: 'entry',
         corejs: { version: 3, proposals: false },
+        exclude: ['transform-regenerator'],
```
Comment (nickytonline) [inline babel.config.js]: This removes polyfills for JavaScript [generators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Generator), [async](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) functions and the [await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await) operator. The version of nodejs we have for running tests support these natively now. The same goes for all the browsers we support. cc: @benhalpern just checking if it's OK to enable this for the tests and application. If not, I can set it to just the `test` environment in the Babel configuration file.

Comment (juliannatetreault) [reply babel.config.js]: Thanks for the explanation! I was just about to ask what is being excluded here and why. 😂

Comment (nickytonline) [reply babel.config.js]: To add a bit more context to this @benhalpern, we currently are not using async/await in the application portion of the codebase, so this just opens the door for people to use it without polyfills (smaller bundles).

Comment (rhymes) [reply babel.config.js]: @nickytonline so excited about this! I think it deservers a heads up to the othe devs, we can finally use `async/await` :D

**Pull request merged**